locksmith/migrations/20220420120142-add-recipients-field-to-charge.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+module.exports = {
```

Comment:
I think we should not use "charges" anymore, but that's fine for this PR.

Reply:
We do use it to save the details so I adapted it.